File: WalkingMaps/WalkingMaps/Snippets/Photo.cs
using System;

namespace WalkingMaps.Entities
{
    public class Photo : IEntityBase
    {

        public int Id { get; set; }      
        public string Title { get; set; }
        public string Uri { get; set; }      
        public int SightId { get; set; }
        public virtual Sight Sight { get; set; }
        public string UserId { get; set; }
        public DateTime DateUploaded { get; set; }
    }
}

File: WalkingMaps/WalkingMaps/Snippets/photo.ts
export class Photo {
    Id: number;
    Title: string;
    Uri: string;
    SightId: number;
    SightTitle: string;
    UserId: string;
    DateUploaded: Date

    constructor(id: number,
        title: string,
        uri: string,
        sightId: number,
        sightTitle: string,
        userId: string,
        dateUploaded: Date) {
        this.Id = id;
        this.Title = title;
        this.Uri = uri;
        this.SightId = sightId;
        this.SightTitle = sightTitle;
        this.UserId = userId;
        this.DateUploaded = dateUploaded;
    }
}